src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -23,7 +23,17 @@ const debugSpy = jest.spyOn(core, 'debug').mockImplementation();
 const getInputSpy = jest.spyOn(core, 'getInput').mockImplementation();
 
 beforeEach((): void => {
-  getInputSpy.mockReturnValue('SQUASH');
+  getInputSpy.mockImplementation((name: string): string => {
+    if (name === 'MERGE_METHOD') {
+      return 'SQUASH';
+    }
+
+    if (name === 'PRESET') {
+      return 'DEPENDABOT_MINOR';
+    }
```

Comment:
Now that the code path checking the `PRESET` is being exercised, this value must be mocked out as well.